react/game/src/app/index.js

```diff
@@ -13,16 +14,17 @@ class App extends Component {
   };
 
   loginProps = () => {
-    if (this.state.loggedin) return <Game />;
+    if (this.state.loggedin) return <Redirect to="/Game" />;
     return <Login redir={this.setRedirect} />;
   };
 
   render() {
     return (
       <Router basename={this.props.path}>
-        <Fragment>
+        <Switch>
           <Route exact path="/" component={this.loginProps} />
```

Comment:
This is a bad use of `Switch`. U must pass in `component` props just the `Login` component. Let that logic of the function `loginProps` make it the `PrivateRoute` component